src/MessageReader.ts

```diff
@@ -4,7 +4,6 @@
 // found in the LICENSE file in the root directory of this source tree.
 // You may not use this file except in compliance with the License.
 
-import int53 from "int53";
```

Comment:
Was wondering if we can now remove this dep, but I see it's also used in BagReader and record.ts

Reply:
Good callout. I've removed it and updated those callsites with the builtin equivalent apis.